docs/content/getting-started/immutable-models.mdx

```diff
@@ -61,10 +61,10 @@ const time = ulid.decodeTime(id);
   description="Learn more about modeling and production usage in {ProductName}."
   relatedLinks={[
     {
-      title: 'Configuration Language',
-      description: 'Learn about the {ProductName} Configuration Language.',
-      link: '../configuration-language',
-      id: '../configuration-language',
+      title: 'Modeling Language',
+      description: 'Learn about the {ProductName} Modeling Language.',
```

Comment:
Elsewhere we seem to use sentence case, but here we use title case. I think sentence case is preferred, what do you think?

Reply:
Ideally we would pick one strategy and make it consistent everywhere, but let me know if I'm missing some nuance here.

Reply:
Section titles are going to be sentence case, yes. My plan is to slightly modify our [current style guide](https://oktawiki.atlassian.net/wiki/spaces/DOCS/pages/2544472407/Auth0+Docs+Style+Guide) for FGA purposes.